usaspending_api/references/management/commands/load_download_static_data.py

```diff
@@ -14,6 +14,7 @@ class Command(BaseCommand):
     def handle(self, *args, **options):
 
         logger = logging.getLogger('console')
+        JobStatus.objects.all().delete()
```

Comment:
What does this do? Seems a bit of a rogue line of code to be here without a comment explaining it (or I just don't the context of this management command).

Reply:
This line allows the static data loading script to be "stateless" i.e. the script can be run over and over and the table will always be left in the same state.